app/build.gradle

```diff
@@ -27,8 +27,8 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile project(':lock-facebook')
-    compile 'com.auth0.android:lock:2.4.0'
+    compile 'com.auth0.android:lock:2.5.0'
     compile 'com.github.bumptech.glide:glide:3.7.0'
-    compile 'com.android.support:support-v4:25.1.0'
-    compile 'com.android.support:appcompat-v7:25.1.0'
+    compile 'com.android.support:support-v4:25.0.0'
```

Comment:
ok. but obviously it wasn't like that before and it still worked, right?
there as another solution to this but we're not using it anywhere yet IMHO. I don't remember the name, `transitive`?
https://docs.gradle.org/current/userguide/dependency_management.html#sub:version_conflicts